website/source/api/agent.html.md

```diff
@@ -566,3 +566,77 @@ fields:
 
 - `Offset` - Offset is the offset into the stream.
 
+## Agent Runtime Profiles
+
+This endpoint is the equivalent of Go's /debug/pprof endpoint but is protected
+by ACLs and supports remote forwarding to a client node or server. See the
+[Golang documentation](https://golang.org/pkg/runtime/pprof/#Profile) for a list of available profiles.
+
+| Method | Path                          | Produces                   |
+| ------ | ----------------------------- | -------------------------- |
+| `GET`  | `/agent/pprof/cmdline`        | `text/plain`               |
+| `GET`  | `/agent/pprof/profile`        | `application/octet-stream` |
+| `GET`  | `/agent/pprof/trace`          | `application/octet-stream` |
+| `GET`  | `/agent/pprof/<pprof profile>`| `application/octet-stream` |
+
+The table below shows this endpoint's support for
+[blocking queries](/api/index.html#blocking-queries) and
+[required ACLs](/api/index.html#acls).
+
+| Blocking Queries | ACL Required  |
+| ---------------- | ------------- |
+| `NO`             | `agent:write` |
+
+### Default Behavior
+
+The default behavior of this endpoint complex but seeks to maximize security,
+backward compatibility, and still allow debug access by default when possible. 
+The table below outlines the different scenarios which will enable or disable
+the endpoint.
```

Comment:
Your text isn't bad, but here's another stab at it. I'm ok with shipping either.

> This endpoint is enabled whenever ACLs are enabled unless [`enable_debug`][enable_debug] is explicitly disabled. Due to the potentially sensitive nature of data contained in profiles as well as their significant performance impact, the agent/pprof endpoint is protected by a high level ACL: `agent:write`. For these reasons it is recommended to leave `enable_debug` unset and only use the ACL-protected endpoints.
>
> The following table explains when each endpoint is available: